CONTRIBUTING.md

```diff
@@ -6,8 +6,8 @@ We'd love to accept your patches! Before we can take them, we have to jump a cou
 
 Please fill out either the individual or corporate Contributor License Agreement (CLA).
```

Comment:
I can't select the line above, but would you mind adjusting it to:
```markdown
[legal hurdles](https://cla.developers.google.com/about)
```

Reply:
Thanks. For the reference: #363 fix this @briandealwis